OneSchedule.Updater/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule.Updater
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var configuration = new ConfigurationBuilder()
+                .AddUserSecrets<Program>()
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            var apiKey = configuration["ApiKey"];
+            var uri = configuration.GetSection("RequstUri").Value;
+            var bot = new TelegramBotClient(apiKey);
+            var offset = 0;
+
+            var client = new HttpClient();
+
+            await bot.SetWebhookAsync("");
```

Comment:
If this is an approach to delete webhook from Telegram then use string.Empty instead of quotes